platforms/documentation/docs/src/snippets/plugins/dependenciesBlock/groovy/buildSrc/src/main/java/com/example/ExampleDependencies.java

```diff
@@ -0,0 +1,14 @@
+package com.example;
+
+import org.gradle.api.artifacts.dsl.Dependencies;
+import org.gradle.api.artifacts.dsl.DependencyCollector;
+
+/**
+ * Custom dependencies block for the example plugin.
+ */
+public interface ExampleDependencies extends Dependencies {
+    /**
+     * Dependency scope called "implementation"
+     */
+    DependencyCollector getImplementation();
```

Comment:
🤔 Maybe it would be better to use a distinct name for this example configuration, instead of `implementation`, which is very common and easy to confuse with the `implementation` from the Java plugin world.

Reply:
I very much want to avoid this being called the same thing as the configuration or using an awkward name to avoid confusion with something else.  The point of this name is to be something meaningful to the plugin.  Since this plugin doesn't do anything, I'm not sure what a better name would be.

Reply:
Requirements?  Libraries?  OptionalTools?  ExtraTools?  Even `getNeededForImplementation` or something.